sound/soc/sof/loader.c

```diff
@@ -203,6 +209,12 @@ static int load_modules(struct snd_sof_dev *sdev, const struct firmware *fw)
 	module = (struct snd_sof_mod_hdr *)((u8 *)(fw->data) + sizeof(*header));
 	remaining = fw->size - sizeof(*header);
 	for (count = 0; count < header->num_modules; count++) {
+		/* check for wrap */
```

Comment:
remaining can already wrap two lines above?

Reply:
and it can also wrap below (line 231)

Reply:
ok